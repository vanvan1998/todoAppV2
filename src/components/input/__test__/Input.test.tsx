import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Input } from '..';

describe('Test Input component', () => {
  test.only('should match its snapshot', async () => {
    const component = render(
      <Input
        title='email'
        value={'email'}
        onChange={jest.fn()}
        styles={{ marginBottom: 16, marginTop: 60 }}
        placeholder='Enter your email...'
      />
    );

    expect(component).toMatchSnapshot();
  });
});
