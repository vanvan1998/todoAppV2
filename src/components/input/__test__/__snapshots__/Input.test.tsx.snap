// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Input component should match its snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        style="margin-bottom: 16px; margin-top: 60px;"
      >
        <div
          class="sc-gueXAH gRHGwY"
        >
          email
        </div>
        <input
          aria-label="email"
          class="sc-gLDymS jWEMcX"
          placeholder="Enter your email..."
          type="text"
          value="email"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      style="margin-bottom: 16px; margin-top: 60px;"
    >
      <div
        class="sc-gueXAH gRHGwY"
      >
        email
      </div>
      <input
        aria-label="email"
        class="sc-gLDymS jWEMcX"
        placeholder="Enter your email..."
        type="text"
        value="email"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
